### DISCUSSION TITLE
Short URLs for latest updates for macOS `.pkg`

### DISCUSSION THREAD

**Discussion by lordofpipes** [microsoft/openjdk#404] [General]
I'm writing support documentation for third party Minecraft launchers, which often requires manually downloading OpenJDK. I'd be interested in having the macOS `.pkg` downloads included as one of the short URLs. Currently only the `.tar.gz` and links to specific versions are available.

**Comment by brunoborges** [comment]
For PKG, you can automate by using Homebrew.
